.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
     strategy:
       matrix:
-        go-version: [1.17, 1.18]
+        go-version: ["1.19", "1.20"]
```

Comment:
Can be `oldstable` and `stable` respectively.

Reply:
Perfect :)